src/lib.rs

```diff
@@ -140,12 +179,13 @@ async fn fetch_program(
 ) -> Result<(), Box<dyn std::error::Error>> {
     let so_filename = format!("spl_{}-{}.so", name.replace('-', "_"), version);
     let so_path = solana_root_path.join(&so_filename);
+    let so_name = format!("spl_{}.so", name.replace('-', "_"));
```

Comment:
ya it's a little weird since the `GenesisProgram` name has hyphens. But for actually storing the file, we use underscores. I'll try and make it a bit clearer

Reply:
Sorry, I meant that more as "sorry that we made this confusing for you" :sweat_smile:

Reply:
e.g.: https://github.com/solana-labs/solana-program-library/releases/download/token-2022-v1.0.0/spl_token_2022.so uses the name `token-2022` and `token_2022`

Reply:
ohhhhh haha ok nvm lol